docs/src/osdk/reference/manifest.md

```diff
@@ -32,58 +32,103 @@ Below, you will find a comprehensive version of
 the available configuration options in the manifest.
 
 ```toml
-kcmd_args = ["init=/bin/busybox", "path=/usr/local/bin"] # <1>
-init_args = ["sh", "-l"] # <2>
-initramfs="./build/initramfs.cpio.gz" # <3>
-
-[boot]
-loader = "grub" # <4>
-protocol = "multiboot2" # <5>
-grub-mkrescue = "/usr/bin/grub-mkrescue" # <6>
-ovmf = "/usr/bin/ovmf" # <7>
-
-[qemu]
-path = "/usr/bin/qemu-system-x86_64" # <8>
-machine = "q35" # <9>
-args = [ # <10>
-    "-enable-kvm",
-    "-m 2G", 
-    "-device virtio-keyboard-pci,disable-legacy=on,disable-modern=off"
-] 
+[project] # <1>
+type = "kernel"
+
+[run] # <2>
+kcmd_args = ["init=/bin/busybox", "path=/usr/local/bin"] # <3>
+init_args = ["sh", "-l"] # <4>
+initramfs="./build/initramfs.cpio.gz" # <5>
+bootloader = "grub" # <6>
+boot_protocol = "multiboot2" # <7>
+grub-mkrescue = "/usr/bin/grub-mkrescue" # <8>
+ovmf = "/usr/bin/ovmf" # <9>
+qemu_path = "/usr/bin/qemu-system-x86_64" # <10>
+qemu_args = [ # <11>
+    "-machine q35,kernel-irqchip=split",
+    "-cpu Icelake-Server,+x2apic",
+    "-m 2G",
+]
+
+[test] # <2>
+bootloader = "qemu"
+qemu_args = [ # <10>
+    "-machine q35,kernel-irqchip=split",
+    "-cpu Icelake-Server,+x2apic",
+    "-m 2G",
+]
+
+['cfg(arch="x86_64", schema=microvm)'.run] # <12>
```

Comment:
`schema`s are mutually exclusive candidates for doing an action.

I have evaluated some candidates for this name:
 - `select`/`selection`, which is a verb. It may sounds well in CLI but not in the case of a written cfg;
 - `profile`, which is really decent. But it conflicts with the build profile and would incur misunderstandings;
 - `feature`, the original proposition. But it implies that the values would be combined if multiple features are enabled. If we support `feature` in the future, then `schema` and `feature` should coexist.